corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -153,6 +154,28 @@ <h4 class="modal-title">{% trans "Create a new Case Type" %}</h4>
           <h3 data-bind="text: $root.activeCaseType()" style="display: inline-block;"></h3>
           <span data-bind="visible: $root.isActiveCaseTypeDeprecated()" class="deprecate-case-type hidden label label-warning" style="display: inline-block;">{% trans "deprecated" %}</span>
         </div>
+        <div class="alert alert-warning" data-bind="visible: showCasePropertyWarning()">
```

Comment:
This looks very similar to what we have in case management page, I didn't do a detailed comparison. But if they're the same, does it make sense to make it a partial?

Reply:
Ended up making the partial work! [8ee6103](https://github.com/dimagi/commcare-hq/pull/36588/commits/8ee6103996bd0483f76a69ba9922bca8006bd489)